client/gulpfile.js

```diff
@@ -28,6 +29,14 @@ const sources = {
     distPath: '../dist/assets/js/',
     all: 'js/**/*.js'
   },
+  jsHead: {
```

Comment:
Would this make more sense as a (e.g. `head`) key on the `sources.js` object? Or maybe it does make more sense to keep them separate.